components/post_view/post/post.jsx

```diff
@@ -259,7 +259,7 @@ export default class Post extends React.PureComponent {
                         />
                     </span>
                 );
-            } else if (isSystemMessage) {
+            } else if (isSystemMessage || post.type === Posts.POST_TYPES.COMBINED_USER_ACTIVITY) {
```

Comment:
Wouldn't `isSystemMessage` be true for a combined user activity post?

Reply:
Ah, you're right.